pennylane/measure.py

```diff
@@ -96,3 +118,28 @@ def __getattr__(self, name):
 Args:
     op (Observable): a quantum observable object
 """
+
+
+def var(op):
+    r"""Variance of the supplied observable.
+
+    Args:
+        op (Observable): a quantum observable object
+    """
+    if not isinstance(op, Observable):
+        raise QuantumFunctionError(
+            "{} is not an observable: cannot be used with var".format(op.name)
+        )
+
+    if QNode._current_context is not None:
+        # delete operations from QNode queue
+        QNode._current_context.queue.remove(op)
```

Comment:
only because the QNode currently maintains two separate queues - one for operations, and one for observables. So we need to manually move it to the correct queue. This is something that should be fixed with a QNode refactor.

Reply:
Note that `QNode._append_op(op)` is responsible for putting `op` in either `self.ev` or `self.queue`, hence why it looks like I'm not really doing anything here.